ops/README.md

```diff
@@ -30,10 +30,19 @@ docker-compose \
 
 Optionally, run a verifier along the rest of the stack.
 ```
-docker-compose up --scale verifier=1 \
+docker-compose up --scale \
+    verifier=1 \
     --build --detach
 ```
 
+Do the same to run an L2 DTL and replica combo. Note that the L2 DTL is a requirement for the replica!
+```
+docker-compose up \
+    --scale l2_dtl=1 --scale replica=1 \
+    --build --detach
+```
```

Comment:
as of now this still doesn't work from scratch when sequencer is at block 0, since we set `highestSyncedL2BlockNumber` to be 1 instead of 0 in the dtl [here](https://github.com/ethereum-optimism/optimism/blob/245136f14afed2a731a5ca5f9b5b071a3131961b/packages/data-transport-layer/src/services/l2-ingestion/service.ts#L76-L79). this triggers a [startBlock > endBlock error](https://github.com/ethereum-optimism/optimism/blob/245136f14afed2a731a5ca5f9b5b071a3131961b/packages/data-transport-layer/src/services/l2-ingestion/service.ts#L140-L149) and prevents the l2_dtl from syncing from scratch, but works during the integration tests since we submit more transactions

Reply:
Should we be setting it to 0 instead?

Reply:
think we'll run into a genesis block error if we do that https://github.com/ethereum-optimism/optimism/pull/981#discussion_r643538928 might consider just changing the sleep condition to `>=` [here](https://github.com/ethereum-optimism/optimism/blob/245136f14afed2a731a5ca5f9b5b071a3131961b/packages/data-transport-layer/src/services/l2-ingestion/service.ts#L89)

Reply:
This is definitely a bug in the DTL, we have had these sorts of problems in the past. A fix was added here, the `-1`

https://github.com/ethereum-optimism/optimism/blob/5e3c5d1c65ef3352947989ba9f51a1047bf7ba0c/packages/data-transport-layer/src/services/server/service.ts#L230

Not sure if that was the most ideal solution but it worked at the time, perhaps we can make another similar change to fix the problem?

Reply:
put in this fix: https://github.com/ethereum-optimism/optimism/pull/981/files#diff-f52a0d7bb7b1308354b3ee1e2099fba24225cceb77b8964638d239375ab10235R89-R93